beets/plugins.py

```diff
@@ -367,7 +297,7 @@ def load_plugins(names: Sequence[str] = ()) -> None:
                         isinstance(obj, type)
                         and issubclass(obj, BeetsPlugin)
                         and obj != BeetsPlugin
-                        and obj != MetadataSourcePlugin
```

Comment:
I'm seeing this

```
** error loading plugin importreplace:
Traceback (most recent call last):
  File "/home/sarunas/repo/beets/beets/plugins.py", line 298, in load_plugins
    and issubclass(obj, BeetsPlugin)
  File "/home/sarunas/.local/share/pyenv/versions/3.9.20/lib/python3.9/abc.py", line 123, in __subclasscheck__
    return _abc_subclasscheck(cls, subclass)
TypeError: issubclass() arg 1 must be a class
```

Reply:
That's the weird bit. I don't get this error on `master` but I get it here 😁

Reply:
Still weird, the only change I introduced here was the check for ABC classes and skipping them. 

I added this line with the hope it fixes the issue:
`and inspect.isclass(obj)`

Reply:
See my comments under https://github.com/beetbox/beets/pull/5739, maybe it's relevant. 

> I added this line with the hope it fixes the issue:
and inspect.isclass(obj)
If so, I think you can remove the `isinstance(obj, type)` check